grpc/pb-marshalling_test.go

```diff
@@ -100,11 +100,11 @@ func TestChallenge(t *testing.T) {
 func TestValidationRecord(t *testing.T) {
 	ip := net.ParseIP("1.1.1.1")
 	vr := core.ValidationRecord{
-		Hostname:          "host",
+		Hostname:          "exampleA.com",
 		Port:              "2020",
 		AddressesResolved: []net.IP{ip},
 		AddressUsed:       ip,
-		URL:               "url",
+		URL:               "http://exampleA.com",
```

Comment:
This URL is http and doesn't have an explicit port, which means the port should be 80, but it's still 2020. Same applies below.